src/app/leverage/layout.tsx

```diff
@@ -12,14 +12,14 @@ type LayoutProps = {
 
 export default function Layout({ children }: LayoutProps) {
   return (
-    <ProvidersLite>
+    <Providers>
       <div className='dark bg-[#141E1F]'>
         <Header />
         <LeverageProvider>
           <main>{children}</main>
         </LeverageProvider>
         <Footer />
       </div>
-    </ProvidersLite>
+    </Providers>
```

Comment:
Has to be used for chakra to work correctly (modals are still chakra) - styles mostly updated to use tailwind only tho